llvm/test/tools/llvm-objcopy/ELF/prefix-symbols-remove.test

```diff
@@ -0,0 +1,78 @@
+# RUN: yaml2obj %s -o %t
+# RUN: llvm-objcopy --prefix-symbols-remove __pf_ %t %t2
+# RUN: llvm-readobj --symbols %t2 | FileCheck %s
```

Comment:
llvm-readelf will provide a more compact output, making it easier to see what's actually interesting about this.

Reply:
Will rewrite this (as well as `prefix-symbols.test`) in a follow-up change.

Reply:
Please don't defer improvements to new tests until a separate change. Please fix this test in this PR. You can address the existing test in a separate PR: the two are not tied together.